test/common.js

```diff
@@ -598,8 +598,7 @@ exports.WPT = {
     try {
       fn();
     } catch (err) {
-      if (err instanceof Error)
-        err.message = `In ${desc}:\n  ${err.message}`;
+      console.error(`In ${desc}:`);
```

Comment:
Hmm, yes, to patch the error message we would have to extend the Error class, something like:

```js
class WPTError extends Error {
  constructor(desc, err) {
    super(`In ${desc}, ${err.name}: ${err.message}`);
    Error.captureStackTrace(this, WPTError);
  }
}

throw new WPTError(desc, err);
```

But then `console.error(`In ${desc}:`)` works too, though we would see two error traces this way?